experimental/framework/fluid-static/CHANGELOG.md

```diff
@@ -13,6 +21,65 @@
     - [[BREAKING] `createDataObject` is replaced by `create`](#breaking-createdataobject-is-replaced-by-create)
       - [Example](#example-3)
 
+## 0.39.0
+
+### [BREAKING] 'FluidStatic' has separate implementations for each service
```

Comment:
Probably good enough here to simply say FluidStatic has moved to independent Server-client packages that will define their own paradigm.

See... (link to tinylicious-client README) for details on Tinylicious client.

Reply:
Updated with link to tinylicious-client README